### DISCUSSION TITLE
I make changes to a file, press commit and when I go back to the file it has not saved

### DISCUSSION THREAD

**Discussion by sophiemaricq** [community/community#63934] [New to GitHub]
Hello! 

1. I have made some updates to the content on a page, pressed the commit button and then when I go back to the page itself the changes haven't been made and I am back to square one. 
2. How can I see the changes on the website before publishing?
3. Where can I make the website go live?

Many thanks in advance

**Comment by oftharsh** [comment]
If you like it please mark it as answer.

It sounds like you're working with a version control system like Git and making changes to a webpage. Here's a step-by-step guide to help you see changes before publishing and making your website go live:

1. **Local Development:**
   - Clone your Git repository to your local machine using `git clone`.
   - Make changes to the webpage content using a code editor.
   - Test the changes locally by opening the webpage in a web browser from your local directory.

2. **Commit Changes:**
   - Use `git status` to see the changes you've made.
   - Stage the changes using `git add .` (to stage all changes) or `git add filename` (to stage specific files).
   - Commit the changes using `git commit -m "Descriptive commit message"`.

3. **Push to Remote:**
   - Push the committed changes to your remote repository using `git push origin branch-name`.

4. **GitHub Pages (Assuming you're using GitHub):**
   - If you're using GitHub, you can use GitHub Pages to host your website.
   - Go to your repository on GitHub.
   - Click on "Settings" tab.
   - Scroll down to the "GitHub Pages" section.
   - Choose the branch you want to publish (usually `main` or `master`).
   - Click "Save".
   - Your website will be live at `https://username.github.io/repository-name`.

5. **Check Live Changes:**
   - After a few moments, your changes should be live on your GitHub Pages URL.
   - Open the URL in a web browser to see the live version of your website.

6. **Domain Name (Optional):**
   - If you have a custom domain, you can configure it to point to your GitHub Pages URL for a professional look.

Remember, there might be a delay before your changes go live due to caching. If the changes are not visible immediately, try clearing your browser cache or waiting for a while.

Also, ensure that you're committing and pushing the changes to the correct branch that is linked to your GitHub Pages setup.

Always follow best practices of version control and website deployment to avoid any issues.

**Comment by sophiemaricq** [comment]
Thanks so much!
